src/v2/BaseLoginRouter.js

```diff
@@ -238,6 +241,9 @@ export default Router.extend({
       ColorsUtil.addStyle(colors);
     }
 
+    // Show before initial render
+    this.show();
```

Comment:
If render fails due to some error, then there is no feedback now to enduser ? They will just see the white screen ?

Reply:
No, user will see error, bacause `startLoginFlow` is being caught
https://github.com/okta/okta-signin-widget/blob/cb805a3dda39ffd61366180f6a6006aedce16b11/src/v2/BaseLoginRouter.js#L227-L228

Loading language is not handling rejections
https://github.com/okta/okta-signin-widget/blob/cb805a3dda39ffd61366180f6a6006aedce16b11/src/v2/BaseLoginRouter.js#L217
https://github.com/okta/okta-signin-widget/blob/cb805a3dda39ffd61366180f6a6006aedce16b11/src/util/LanguageUtil.js#L21
In case of error for loading language before change user will see infinite spinner, not he will see nothing